Mlem/API/Models/Posts/APIPostReport.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  APIPostReport.swift
+//  Mlem
+//
+//  Created by Jake Shirley on 7/1/23.
+//
+
+import Foundation
+
+// lemmy_db_schema::source::post::PostReport
+struct APIPostReport: Decodable {
+    let id: Int
+    let creatorId: Int
+    let postId: Int
+    let originalPostName: String
+    let originalPostUrl: String?
```

Comment:
This can be a `URL?`, but this is also alright

Reply:
Ah good catch.